Formula/link-grammar.rb

```diff
@@ -30,15 +31,14 @@ class LinkGrammar < Formula
 
   def install
     ENV["PYTHON_LIBS"] = "-undefined dynamic_lookup"
-    inreplace "bindings/python/Makefile.am",
-      "$(PYTHON_LDFLAGS) -module -no-undefined",
-      "$(PYTHON_LDFLAGS) -module"
-    system "autoreconf", "-fiv"
-    system "./configure", "--disable-debug",
-                          "--disable-dependency-tracking",
-                          "--prefix=#{prefix}",
-                          "--with-regexlib=c"
-    system "make", "install"
+    inreplace "bindings/python/Makefile.am", "$(PYTHON_LDFLAGS) -module -no-undefined",
+                                             "$(PYTHON_LDFLAGS) -module"
+    system "autoreconf", "--verbose", "--install", "--force"
+    system "./configure", *std_configure_args, "--with-regexlib=c"
+
+    site_packages = prefix/Language::Python.site_packages("python3")
+    system "make", "install", "pythondir=#{site_packages}",
+                              "pyexecdir=#{site_packages}"
```

Comment:
Only overriding values at install-time seemed to work (either `pythondir` like above, or `PYTHON_PREFIX=#{prefix}`) while environment variables & configure-time flags didn't help.

May be better to add a `configure` flag to override this value so all generated Makefiles are set to requested path. I don't know what is recommended way to do this, but some examples from other formulae include:
- graph-tool has configure flag `--with-python-module-path`
- bind has configure  flag `--with-python-install-dir`
- stp has cmake variable `PYTHON_LIB_INSTALL_DIR`